.github/workflows/test.yaml

```
@@ -21,6 +21,9 @@ on:
       python_runtimes:
         description: 'Included python runtimes in the build'
         required: false
+      version_cache:
+        description: 'Content of the agent-version.cache file'
```

```suggestion
        description: 'Base 64-encoded content of the agent-version.cache file'
```

+ do the same in the `macos.yaml` workflow.